configs/espnetv1/README.md

```diff
@@ -0,0 +1,11 @@
+# ESPNet: Efficient Spatial Pyramid of Dilated Convolutions for Semantic Segmentation
+
+
+## Reference
+[1] Mehta S ,  Rastegari M ,  Caspi A , et al. ESPNet: Efficient Spatial Pyramid of Dilated Convolutions for Semantic Segmentation[J]. Springer, Cham, 2018.
```

Comment:
Plz change this quote to Chicago format, which includes their full name.
you can refer to this: https://github.com/PaddlePaddle/PaddleSeg/blob/develop/docs/pr/pr/style_cn.md#%E4%B8%89%E6%96%B0%E5%A2%9E-pr-checklist

Reply:
fixed